ExilenceNextApp/src/store/domains/profile.ts

```diff
@@ -99,13 +99,8 @@ export class Profile {
   }
 
   @computed
-  get latestSnapshotItemCount() {
-    if (this.snapshots.length === 0) {
-      return 0;
-    }
-    return this.snapshots.flatMap(s =>
-      s.stashTabSnapshots.flatMap(sts => sts.items)
-    ).length;
+  get latestSnapshotItemCount(): number {
+    return this.filteredItems.length;
   }
```

Comment:
We should include stacksize too when we count, in my opinion. So if we have a stack of 40 chaos, it counts as 40 items.